src/main/kotlin/com/example/olaf/blogtutorial/Entities.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.olaf.blogtutorial
+
+import java.time.LocalDateTime
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+
+@Entity
+class Article(
+    var title: String,
+    var headline: String,
+    var content: String,
+    @ManyToOne var author: User,
+    var slug: String = title.toSlug(),
+    var addedAt: LocalDateTime = LocalDateTime.now(),
+    @Id @GeneratedValue var id: Long? = null
+)
+
+@Entity
+class User(
+    var login: String,
+    var firstname: String,
+    var lastname: String,
```

Comment:
```suggestion
    var firstName: String,
    var lastName: String,
```
이렇게 해야 녹색 밑줄이 없어질것 같아요.
지금은 in word 인가 하는 철자 맞지 않는 다는 워닝을 주고 있을 것 같아요.

Reply:
넵 수정했습니다 😄